mylar/req_test.py

```diff
@@ -78,7 +78,11 @@ def check_the_pip(self):
                                 p_version = str(line[p_test+2:]).strip()
                                 if p_version == '':
                                     continue
-                                self.req_list.append({'module': p_mod, 'version': p_version, 'arg': p_arg})
+                                if p_mod == 'requests[socks]':
+                                    self.req_list.append({'module': 'requests', 'version': '2.22', 'arg': '>='})
```

Comment:
Hardcoding version numbers? No way to reference requirements.txt current values?

Reply:
Yeah was pretty ugly right? 

Because we're importing socks as part of requests it only holds requests version number.  And when I did on my ancient system is installed 1.7.1, so I just took a lower value in case some bloke is running older hardware than I am haha. 

The other alternative is to just add ``PySocks`` to the requirements.txt and change ``requests[socks``] back to ``requests``.